### PR TITLE
[SLDEV-4576] End execution as post build step

### PR DESCRIPTION
**Pull request by RoniSegal**

### CODE REVIEW

File: src/main/java/io/sealights/plugins/sealightsjenkins/EndExecutionPostBuildStep.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import io.sealights.plugins.sealightsjenkins.utils.Logger;
+import io.sealights.plugins.sealightsjenkins.utils.PropertiesUtils;
+import io.sealights.plugins.sealightsjenkins.utils.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.export.Exported;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class EndExecutionPostBuildStep extends Recorder {
```
Comment (nadavye) [inline src/main/java/io/sealights/plugins/sealightsjenkins/EndExecutionPostBuildStep.java:25]: Please document all the new classes.

Comment (RoniSegal) [reply src/main/java/io/sealights/plugins/sealightsjenkins/EndExecutionPostBuildStep.java:30]: done

File: src/test/java/io/sealights/plugins/sealightsjenkins/EndExecutionPostBuildStepTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    @Test
+    public void perform_bsidNull_shouldNotRunCommand() throws IOException, InterruptedException {
+        EndExecutionPostBuildStep endExecutionPostBuildStep = createInstance(null,
+                null);
+        EndExecutionPostBuildStep spy = spy(endExecutionPostBuildStep);
+
+        spy.perform(build, launcher, listener);
+
+        verify(spy, never()).runCLICommand(eq(build), eq(launcher), eq(listener), any(CommandMode.EndView.class),
+                any(Logger.class), any(LogConfiguration.class), eq(""));
```
Comment (nadavye) [inline src/test/java/io/sealights/plugins/sealightsjenkins/EndExecutionPostBuildStepTest.java:75]: Is there a reason not to use `anyString()` in `perform_bsidNull_shouldNotRunCommand` and `perform_bsidNotResolved_shouldNotRunCommand` as the lab id?

Comment (RoniSegal) [reply src/test/java/io/sealights/plugins/sealightsjenkins/EndExecutionPostBuildStepTest.java:75]: No preticular reason, I'm agree that `anyString()` is more correct

**Pull request merged**